Added FAQ on mounting volumes to VMs (#28)
rancher-sandbox/docs.rancherdesktop.io: docs/faq.md

```
Visit [http://127.0.0.1:9000/dashboard/](http://127.0.0.1:9000/dashboard/) in your browser to view the Traefik dashboard.

**Q: Does file sharing work similarly to Docker Desktop? Do I have to do any additional configuration to mount volumes to VMs?**

> **A:** Currently, the following directories are shared by default: `/Users/$USER` on macOS, `/home/$USER` on Linux, and `/tmp/rancher-desktop` on both. For Windows, these files are automatically shared via WSL2. We will update the community as more mounted directories are added to this list.

Reviewer: ```suggestion
**A:** Currently, the following directories are shared by default: `/Users/$USER` on macOS, `/home/$USER` on Linux, and `/tmp/rancher-desktop` on both. For Windows, all files are automatically shared via WSL2.
```

"these files" refers to the files referred to before, which are macOS and Linux specific. So I think "all files" is better.

I think it is implied that we will update the document as our product changes; it is not specific to this particular question.

Author: Committed as suggested, thanks!
